driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedTestModifications.java

```diff
@@ -474,7 +531,7 @@ public TestApplicator testContains(final String dir, final String fragment) {
          * @param test the individual test's "description" field
          * @return this
          */
-        public TestApplicator test(final String dir, final String test) {
+        public TestApplicator debug(final String dir, final String test) {
```

Comment:
@katcharov I renamed this method as I found it too easy to use thinking it was a matcher.

Reply:
Good call